catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -204,21 +206,21 @@ public static void processEntry(Metacard metacard, String operation, EventAdmin
       LOGGER.debug("Unable to post null metacard.");
     }
 
-    LOGGER.debug("EXITING: {}", methodName);
+    LOGGER.debug(EXITING, methodName);
```

Comment:
I think the following logging messages are more appropriate at the `TRACE` level

Reply:
@bdeining does that apply to every area that's using the EXITING and ENTERING constants?

Reply:
I believe that to be the case : https://codice.atlassian.net/wiki/spaces/DDF/pages/7045131/Log+Levels

Reply:
Just a note that an exit log like this might not always be traced. It will only be traced if we exit successfully. If an exception occurs, no exit trace would be generated and that could be confusing.